[2023-08-16T09:04:54.869+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: rapid_api_v1.load_db scheduled__2023-08-05T00:00:00+00:00 [queued]>
[2023-08-16T09:04:54.877+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: rapid_api_v1.load_db scheduled__2023-08-05T00:00:00+00:00 [queued]>
[2023-08-16T09:04:54.878+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-08-16T09:04:54.879+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-08-16T09:04:54.880+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-08-16T09:04:54.888+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): load_db> on 2023-08-05 00:00:00+00:00
[2023-08-16T09:04:54.894+0000] {standard_task_runner.py:55} INFO - Started process 148 to run task
[2023-08-16T09:04:54.899+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rapid_api_v1', 'load_db', 'scheduled__2023-08-05T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/jobs_etl.py', '--cfg-path', '/tmp/tmp0jl3lfe1']
[2023-08-16T09:04:54.900+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask load_db
[2023-08-16T09:04:54.970+0000] {task_command.py:389} INFO - Running <TaskInstance: rapid_api_v1.load_db scheduled__2023-08-05T00:00:00+00:00 [running]> on host 17c4b1dd2309
[2023-08-16T09:04:55.045+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=rapid_api_v1
AIRFLOW_CTX_TASK_ID=load_db
AIRFLOW_CTX_EXECUTION_DATE=2023-08-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-05T00:00:00+00:00
[2023-08-16T09:04:55.102+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-08-16T09:04:55.113+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=rapid_api_v1, task_id=load_db, execution_date=20230805T000000, start_date=20230816T090454, end_date=20230816T090455
[2023-08-16T09:04:55.153+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-08-16T09:04:55.173+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
